shared/api/network.go

```diff
@@ -152,6 +152,12 @@ type NetworkLease struct {
 	//
 	// API extension: network_leases_location
 	Location string `json:"location" yaml:"location"`
+
+	// Name of the project of the entity related to the hostname
+	// Example: default
+	//
+	// API extension: network_allocations_ovn_uplink
+	Project string `json:"project" yaml:"project"`
```

Comment:
commit message is out of date.

Also have you reviewed where else NetworkLease is used and checked whether they need to be updated with the new Project field?

Reply:
Yes I checked all the places it is used, lucky for me there aren't that many as you can see.
I am still tidying up the tests and commits and then I will ping you for a full review :D